kola/tests/crio/crio.go

```diff
@@ -205,10 +206,10 @@ func crioBaseTests(c cluster.TestCluster) {
 
 // generateCrioConfig generates a crio pod/container configuration
 // based on the input name and arguments returning the path to the generated configs.
-func generateCrioConfig(name string, command []string) (string, string, error) {
-	fileContentsPod := fmt.Sprintf(crioPodTemplate, name, name)
+func generateCrioConfig(podName string, imageName string, command []string) (string, string, error) {
```

Comment:
You can change this to be `podName, imageName string, command []string`

Reply:
Ah, cool...TIL